src/borg/archive.py

```diff
@@ -1976,18 +1977,35 @@ def save(self, archive, target, comment=None, replace_original=True):
             return
         if comment is None:
             comment = archive.metadata.get('comment', '')
-        target.save(comment=comment, stats=target.stats, additional_metadata={
-            # keep some metadata as in original archive:
-            'time': archive.metadata.time,
-            'time_end': archive.metadata.get('time_end') or archive.metadata.time,
-            'cmdline': archive.metadata.cmdline,
-            # but also remember recreate metadata:
-            'recreate_cmdline': sys.argv,
-        })
+
+        # Keep for the statistics if necessary
+        if self.stats:
+            _start = target.start
+
+        if self.timestamp is None:
+            end = archive.metadata.get('time_end') or archive.metadata.time
+            start = archive.metadata.time
+            additional_metadata = {
+                'time': start,
+                'time_end': end,
```

Comment:
as start and end are only used here, maybe directly assign without using these variables, like in the old code?